feat(APP-4139): Update AlertCard component to use children and remove the description property (#442)
aragon/gov-ui-kit: CHANGELOG.md

## [Unreleased]

- **BREAKING**: Update implementation of `AlertCard` component:

>     - Remove `description` property from the `AlertCard` component and prefer passing of children instead

Reviewer: ```suggestion
    - Remove `description` property from the `AlertCard` component in favour of `children` property
```
